site/static/docs/4.3/assets/scss/_subnav.scss

```diff
@@ -7,11 +7,6 @@
     opacity: .25;
   }
 
-  .dropdown-toggle {
```

Comment:
On my iPad so I can’t confirm, but was the removal of all of this intentional? Can’t recall on the `text-decoration`.

Reply:
Yes it is. I noticed the `font-size` was redundant and the only property left in the block was `text-decoration`, which also is redundant (text-decoration is removed by `.btn`)